tools/modules/update.sh

```diff
@@ -16,28 +16,30 @@
 # update_modules.sh updates the repositories for external code (to the precise
 # revisions used for testing).
 #
-# If the single argument `--git_only` is passed, the repositories will be
+# If the single argument `--fetch_only` is passed, the repositories will be
 # updated to the pinned versions without configuring or building them.
 # If the argument `--build_only` is passed, the script will assume that the
 # repositories are at the correct version and will configure and build them.
 
-git_maybe_clone() {
-  local repo="$1"
+wget_copy_archive() {
+  local target="$1"
```

Comment:
Please document this in some wise—"target" is very general.

Reply:
:+1: